### PR TITLE
Add support for reading occupancy and humidity

### PR DESCRIPTION
**Pull request by jamesmulcahy**

### CODE REVIEW

**Comment by jamesmulcahy** [comment]
I have another change for promobee coming, which builds upon this to export metrics for humidity and occupancy.

File: types.go [hunk 1]
```diff
@@ -342,6 +343,34 @@ func (s *RemoteSensor) Temperature() (float64, error) {
...
 }
 
+// Humidity gets the humidity for the sensor if it exists.
+func (s *RemoteSensor) Humidity() (int, error) {
+	if s != nil && len(s.Capability) > 0 {
+		for _, c := range s.Capability {
+			if c.Type == CapabilityTypeHumidity {
+				v, err := strconv.ParseInt(c.Value, 0, 0)
+				if err != nil {
+					return 0.0, err
```
Comment (cfunkhouser) [inline types.go]: Should just be `0` here to reduce confusion, since this function returns `int`.

```suggestion
					return 0, err
```

**Review by cfunkhouser — APPROVED** [review]
Looks good, thanks! One suggestion, see below.

WRT `promobee`, I've not tried to use it for some time. Your mileage may vary. I'd very much _like_ to get it working, but I'm not sure it does right now.

File: types.go (hunk 1 above)
Comment (jamesmulcahy) [reply types.go]: Good spot -- thanks! GitHub's new (to me) "Commit Suggestion" looks like it'll create a separate commit, which seems less than ideal to me, so I fixed this locally and re-pushed.  PTAL!

re: promobee -- it's working great for me.  I've already extended it to plot Humidity, HVAC Status (off/heat/cool), HVAC Operation (cooling, heating, fan status), as well as occupancy status.

I'll send some PRs once I've got this merged (and worked out how to update the vendored egobee in promobee; I'm relatively new to golang vendoring, so any pointers there would be appreciated!)

Comment (cfunkhouser) [reply types.go]: Glad to hear it! You've motivated me to get mine running again. 😃

**Pull request merged**